### PR TITLE
Add onboarding module for automatic ERC20 wrapping

### PR DESCRIPTION
**Pull request by ChristianBorst**

### CODE REVIEW

File: ibcutils/testing/README.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
+# IBC Testing Package
+
+## Components
```
Comment (jkilpatr) [inline ibcutils/testing/README.md:3]: IIRC, you said that we didn't end up using this and went for rust testing suite validation instead. should we remove this testing suite if it's not running?

File: integration_tests/test_runner/src/bin/main.rs [hunk 2]
```diff
@@ -155,6 +159,58 @@ pub async fn main() {
             info!("Start ICA Host test");
             ica_host_happy_path(&contact, &ibc_contact, keys, ibc_keys).await;
             return;
+        } else if test_type == "ONBOARDING_DEFAULT_PARAMS" {
+            start_ibc_relayer(&contact, &ibc_contact, &keys, &ibc_keys).await;
+            info!("Start onboarding default params test");
+            onboarding_default_params(
+                &contact,
+                &ibc_contact,
+                keys,
```
Comment (jkilpatr) [inline integration_tests/test_runner/src/bin/main.rs:168]: there should be a human readable summary of the test goals somewhere, but maybe I'm missing it when looking here.

**Review by jkilpatr — CHANGES_REQUESTED** [review]
How much of this IBC testsuite is running? I think from our discussion that we don't end up running this and can maybe carve several thousand lines out of this PR by not importing code we don't use.

But if it's being used by some test that does run successfully I have qualms about keeping it around.

File: ibcutils/testing/README.md (hunk 1 above)
Comment (ChristianBorst) [reply ibcutils/testing/README.md:3]: As I mentioned, there are some valuable tests using this package, this directory should now only contain alive code.

**Review by jkilpatr — APPROVED** [review]

**Pull request merged**